### ISSUE TITLE
CUBLAS_STATUS_MAPPING_ERROR when train segmentation model

### ISSUE THREAD

**Issue by smarttowel** [NVIDIA/DIGITS#1817]
I try to train image segmentation neural network on my own dataset. My dataset contains only 1 class (and background, of course).
I change num_outputs from 21 to 2 in score_fr and upscore layers, also I change group=2. After that I rename score_fr to score_fr_ and upscore to upscore_. 
When I try to train my network I see "ERROR: Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0) CUBLAS_STATUS_MAPPING_ERROR" message.

Full output:
conv1 needs backward computation.
shift does not need backward computation.
label_label_0_split does not need backward computation.
label does not need backward computation.
data_data_0_split does not need backward computation.
data does not need backward computation.
This network produces output accuracy
This network produces output loss
Network initialization done.
Solver scaffolding done.
Finetuning from /home/pavel/fcn-alexnet-pascal.caffemodel
Ignoring source layer score_fr
Ignoring source layer upscore
Ignoring source layer score_fr
Ignoring source layer upscore
Starting Optimization
Solving
Learning Rate Policy: step
Iteration 0, Testing net (#0)
Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0)  CUBLAS_STATUS_MAPPING_ERROR

What I doing wrong?

**Comment by lukeyeager** [comment]
How did you install Caffe? You're probably going to need to ask over at `NVIDIA/caffe`.

**Comment by smarttowel** [comment]
I simply install digits from nvidia repos via `sudo apt install digits` that requires caffe as dependency.

**Comment by lukeyeager** [comment]
> CUBLAS_STATUS_MAPPING_ERROR
An access to GPU memory space failed, which is usually caused by a failure to bind a texture.
To correct: prior to the function call, unbind any previously bound textures.
http://docs.nvidia.com/cuda/cublas/index.html

That's weird. I'd ask @drnikolaev over at `NVIDIA/caffe`.
